ale/drivers/base.py

```diff
@@ -19,7 +19,7 @@ class Base(ABC):
 
     """
     def __init__(self, label, *args, **kwargs):
-        self.label = pvl.loads(label)
+        self._label = pvl.loads(label, strict=False)
```

Comment:
Tight couple to PVL is probably fine for now. This will be another place to have multi-format support (for example to read an ALE derived JSON header).